backend/api/admin/ohshown_event.py

```diff
@@ -278,13 +278,13 @@ def google_map_link(self, obj):
 
         return format_html(html_template.format(lat=obj.lat, lng=obj.lng))
 
-    @set_function_attributes(short_description="Disfactory Map 連結")
-    def disfactory_map_link(self, obj):
-        disfactory_frontend_domain = os.environ.get(
-            "DISFACTORY_FRONTEND_DOMAIN", "https://disfactory.tw/"
+    @set_function_attributes(short_description="Ohshown Map 連結")
+    def ohshown_map_link(self, obj):
+        ohshown_frontend_domain = os.environ.get(
+            "OHSHOWN_FRONTEND_DOMAIN", "http://ohshown.site:8080/"
```

Comment:
note: we will need to update the corresponding `.env.sample` or `.env` as well

Reply:
Track the update with https://github.com/tai271828/disfactory-backend/pull/33/files#r792700540